Cinebase/Gateways/Container/MoviesContainer.swift

```diff
@@ -9,15 +9,130 @@
 import Foundation
 import RealmSwift
 
-class MoviesContainer {
+class MoviesContainer: LocalContainer {
     
-    func fetchNowPlaying(completion: @escaping (Result<[Movie]>) -> Void) {
+    typealias Model = Movie
+    typealias ModelID = Int
+    
+    /// Name for the realm file.
+    /// Subclasses should override this property to
+    /// target a different container.
+    class var containerName: String {
+        return "Movies"
+    }
+    
+    private static func makeRealm() throws -> Realm {
```

Comment:
what about an init method to pass the container name? i saw that you override the container name in each container.